src/client/pages/HLI/HLIStadspas.tsx

```diff
@@ -294,17 +303,20 @@ function BlockStadspas({ stadspas }: { stadspas: StadspasFrontend }) {
           onClick={() => {
             setIsModalOpen(true);
           }}
-          data-testid="block-stadspas-button"
         >
           Blokkeer deze Stadspas
         </Button>
       )}
-
       <Modal
         title="Weet u zeker dat u uw Stadspas wilt blokkeren?"
         className={styles.BlokkeerDialog}
         isOpen={isModalOpen}
         showCloseButton={false}
```

Comment:
```suggestion
        showCloseButton
```
Als je kunt escapen, dan ook een kruisje :-)

Reply:
Ah, dus dat is de consensus. Soort van dat er ook een 'esc' in de gui zichtbaar moet zijn als je dit voor de keyboard aanbied?